src/middleware/imageUpload.js

```diff
@@ -15,31 +15,36 @@ const uploadToCloudinary = async (filePath) => {
     let mainFolderName = "tusomeniImages"
     let filePathOnCloudinary = `${mainFolderName}/${path.basename(filePath)}`
 
-    return cloudinary.uploader.upload(filePath, {"public_id": filePathOnCloudinary})
-    .then(res => {
-        fs.unlinkSync(filePath) // remove file from local upload folder
-
-        return {
-            message: "Success",
-            url: res.url
-        }
-    }).catch(err => {
-        fs.unlinkSync(filePath);
-        throw new Error("Failed to upload images")
-    })
+    return cloudinary.uploader.upload(filePath, { "public_id": filePathOnCloudinary })
+        .then(res => {
+            fs.unlinkSync(filePath) // remove file from local upload folder
+
+            return {
+                message: "Success",
+                url: res.url
+            }
+        }).catch(err => {
+            fs.unlinkSync(filePath);
+            throw new Error("Failed to upload images")
+        })
 
 }
 
 
 const storage = multer.diskStorage({
     destination: function (req, file, cb) {
-      cb(null, './uploads')
+        cb(null, '')
```

Comment:
The "./uploads" directory is a feature. This is where images are stored temporarily before being uploaded to Cloudinary.